packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -59,7 +60,8 @@ export class ModuleConstructor {
 
   constructor(
     public readonly chainId: number,
-    public readonly accounts: string[]
+    public readonly accounts: string[],
+    public readonly parameters: ModuleParameters = {}
```

Comment:
I think this should get a `ModuleParameters` per module. More like:
```suggestion
    public readonly parameters: {[moduleId: string]: ModuleParameters} = {}
```